Integrations/GoogleCloudStorage/GoogleCloudStorage.py

```diff
@@ -0,0 +1,369 @@
+import traceback
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+from google.cloud import storage
+
+import datetime
+import json
+import re
+import requests
+# from distutils.util import strtobool
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+''' GLOBALS/PARAMS '''
+
+RFC3339_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DEMISTO_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+PROJECT_ID = demisto.params().get('project_id')
+SERVICE_ACCOUNT_JSON = demisto.params().get('service_account_json')
+USE_PROXY = demisto.params().get('use_proxy')
+
+# USERNAME = demisto.params().get('credentials').get('identifier')
+# PASSWORD = demisto.params().get('credentials').get('password')
+# TOKEN = demisto.params().get('token')
+# # Remove trailing slash to prevent wrong URL path to service
+# SERVER = demisto.params()['url'][:-1] \
+#     if (demisto.params()['url'] and demisto.params()['url'].endswith('/')) else demisto.params()['url']
+# # Should we use SSL
+# USE_SSL = not demisto.params().get('insecure', False)
+# # How many time before the first fetch to retrieve incidents
+# FETCH_TIME = demisto.params().get('fetch_time', '3 days')
+# # Service base URL
+# BASE_URL = SERVER + '/api/v2.0/'
+# # Headers to be sent in requests
+# HEADERS = {
+#     'Authorization': 'Token ' + TOKEN + ':' + USERNAME + PASSWORD,
+#     'Content-Type': 'application/json',
+#     'Accept': 'application/json'
+# }
+
+
+def safe_del(dictionary, key):
+    if key in dictionary:
+        del dictionary[key]
+
+
+# Remove proxy if not set to true in params
+if not USE_PROXY:
+    safe_del(os.environ, "HTTP_PROXY")
+    safe_del(os.environ, "HTTPS_PROXY")
+    safe_del(os.environ, "http_proxy")
+    safe_del(os.environ, "https_proxy")
+
+
+''' HELPER FUNCTIONS '''
+
+
+def init_storage_client():
+    cur_directory_path = os.getcwd()
+    credentials_file_name = "{}.json".format(demisto.uniqueFile())
+    credentials_file_path = os.path.join(cur_directory_path, credentials_file_name)
+    with open(credentials_file_path, "w") as creds_file:
+        json_object = json.loads(SERVICE_ACCOUNT_JSON)
+        json.dump(json_object, creds_file)
+    return storage.Client.from_service_account_json(credentials_file_path)
+
+
+def ec_key(path, *merge_by):
```

Comment:
This is great, please just make sure that it works properly for all the calls in this integration